arabic/unpacked/html.js

```diff
@@ -0,0 +1,90 @@
+MathJax.Hub.Config({
+  'HTML-CSS': {
+    undefinedFamily: 'Amiri',
+    styles: {
+      '.mfliph': {
+        'display': 'inline-block !important',
+        '-moz-transform': 'scaleX(-1)',
+        '-webkit-transform': 'scaleX(-1)',
+        '-o-transform': 'scaleX(-1)',
+        'transform': 'scaleX(-1)',
+        '-ms-filter': 'fliph',
+        'filter': 'fliph'
+      },
+      '.mar': {
+        'font-family': 'Amiri !important',
+        'font-style': 'normal !important'
+      }
+    }
+  }
+});
+
+
+MathJax.Hub.Register.StartupHook('mml Jax Ready', function () {
+  MathJax.Hub.Register.StartupHook('HTML-CSS Jax Ready', function () {
+    MathJax.Hub.Register.StartupHook('Arabic TeX Ready', function () {
```

Comment:
The `HTML-CSS Jax Ready` signal is guaranteed to come after the `mml Jax Ready` signal, so there is no need to wait on both (though Kudos for making sure both are ready).

If you end up not having a bunch of separate files, then you probably won't need to the `Arabic TeX Ready` signal, as you can just use the position within the arabic.js file to handle that.  (You will still want a final `Arabic Ready` signal of some sort, so that other extensions can synchronize with yours.)